packages/ripple-binary-codec/src/types/issue.ts

```
@@ -4,24 +4,35 @@ import { BinaryParser } from '../serdes/binary-parser'
+ * Type guard for Issue Object
  */
 function isIssueObject(arg): arg is IssueObject {
   const keys = Object.keys(arg).sort()
-  if (keys.length === 1) {
-    return keys[0] === 'currency'
-  }
-  return keys.length === 2 && keys[0] === 'currency' && keys[1] === 'issuer'
+  const isXRP = keys.length === 1 && keys[0] === 'currency'
+  const isIOU =
+    keys.length === 2 && keys[0] === 'currency' && keys[1] === 'issuer'
+  const isMPT = keys.length === 2 && keys[0] === 'mpt_issuance_id' && keys[1] === 'value'
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)